code-challenges.js

```diff
@@ -17,6 +41,18 @@ var arrayOfWords = ["Apple", "Banana", "Plum", "Cherry", "Kiwi", "Peach"]
 // Expected output: "Apple" "Banana" "Peach"
 
 
+const aFilter = (array) => {
+  // have to filter through the array to make a subset of class
+  let wordFilter = array.filter(value => {
+    //is there a way to make this more dynamic?
```

Comment:
I like that you asked this question because indeed, there very much is a way to do this more dynamically (without hard coding upper and lower case versions of the same letter). What are some js functions you've used to manipulate strings to change casing?